infomaniak-build-tools/kDrive-template.html

```diff
@@ -19,18 +19,28 @@
 <!DOCTYPE html>
 <html>
 <body>
-<h1>Release notes - kDrive 3.6.6</h1>
+<h1>Release notes - kDrive 3.6.7</h1>
+<h2><b>New features</b></h2>
+  <ul>
+    <li>Change the update mechanism to allow progressive delivery of new version</li>
```

Comment:
```suggestion
    <li>Change the update mechanism to allow progressive delivery of a new version</li>
```

Reply:
Done in ee84a9ed3e8c9765067d576481598f818eb5669c